_maps/shuttles/syndicate/syndicate_litieguai.dmm

```diff
@@ -1,989 +1,1108 @@
+"nF" = (
+/obj/machinery/atmospherics/components/unary/vent_scrubber/on/layer4{
 	dir = 8
 	},
-/obj/machinery/door/firedoor/border_only{
-	dir = 4
-	},
-/turf/open/floor/plasteel/grimy,
+/turf/open/floor/carpet/cyan,
 /area/ship/crew)
-"mw" = (
-/obj/machinery/door/airlock/maintenance_hatch{
-	dir = 4;
-	name = "Port Engines"
-	},
-/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
-	dir = 4
+"nI" = (
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2,
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
+/obj/machinery/door/firedoor/border_only{
+	dir = 1
 	},
-/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
-	dir = 4
+/obj/machinery/door/firedoor/border_only,
+/obj/machinery/door/airlock/medical{
+	name = "Treatment Center"
 	},
 /obj/structure/cable{
-	icon_state = "4-8"
+	icon_state = "1-2"
 	},
+/turf/open/floor/plasteel/tech/grid,
+/area/ship/medical)
+"nW" = (
+/obj/structure/cable,
+/obj/effect/turf_decal/trimline/opaque/red/filled/line,
 /obj/machinery/door/firedoor/border_only{
 	dir = 8
 	},
 /obj/machinery/door/firedoor/border_only{
 	dir = 4
 	},
-/turf/open/floor/plating,
-/area/ship/maintenance/port)
-"mG" = (
-/obj/machinery/firealarm/directional/north,
-/obj/structure/sign/warning/gasmask{
-	pixel_x = -32
-	},
-/obj/item/stack/marker_beacon/thirty,
-/obj/item/stack/marker_beacon/thirty,
-/obj/item/stack/marker_beacon/thirty,
-/obj/item/stack/marker_beacon/thirty,
-/obj/structure/rack,
-/obj/item/pickaxe/emergency{
-	desc = "For extracting yourself from rough landings, and getting to the even rougher ones";
-	name = "Medical Retrieval Tool"
+/obj/machinery/power/apc/auto_name/directional/south,
+/obj/machinery/light_switch{
+	dir = 1;
+	pixel_x = 11;
+	pixel_y = -16
 	},
-/obj/item/pickaxe/emergency{
-	desc = "For extracting yourself from rough landings, and getting to the even rougher ones";
-	name = "Medical Retrieval Tool"
+/turf/open/floor/plasteel/white,
+/area/ship/hallway/central)
+"oc" = (
+/obj/effect/turf_decal/corner/opaque/red/full,
+/obj/structure/cable{
+	icon_state = "4-8"
 	},
-/obj/item/pickaxe/emergency{
-	desc = "For extracting yourself from rough landings, and getting to the even rougher ones";
-	name = "Medical Retrieval Tool"
+/turf/open/floor/plasteel/white,
+/area/ship/cargo)
+"ot" = (
+/obj/machinery/door/poddoor/preopen{
+	dir = 8;
+	id = "traumabridge";
+	name = "Window Blast Door"
 	},
-/obj/effect/turf_decal/industrial/outline/red,
-/turf/open/floor/plasteel/tech/grid,
-/area/ship/storage)
-"mO" = (
-/obj/machinery/computer/med_data,
-/turf/open/floor/plasteel/dark,
+/obj/structure/grille,
+/obj/structure/window/reinforced/fulltile/shuttle,
+/turf/open/floor/plating,
 /area/ship/bridge)
-"mP" = (
-/obj/structure/sign/departments/medbay/alt,
-/turf/closed/wall/mineral/titanium/nodiagonal,
-/area/ship/maintenance/port)
-"mQ" = (
-/obj/structure/window/reinforced{
-	dir = 8
+"oA" = (
+/obj/effect/turf_decal/trimline/opaque/red/filled/warning{
```

Comment:
Edit the red decals in the hallway for consistency between each half; warning decals at doorways, solid decals along walls. That, or use your own pattern, just try to make it consistent. There's a few blank tiles in the aft part of the hallway that stick out badly.